AzureMonitorAgent/agent.py

```diff
@@ -524,7 +524,11 @@ def enable():
     the settings provided are incomplete or incorrect.
     Note: enable operation times out from WAAgent at 5 minutes
     """
-    global AMAServiceStartCommand, AMAServiceStatusCommand
+    global AMAServiceStartCommand, AMAServiceStatusCommand, HUtilObject
```

Comment:
fwiw you don't need to `global` the HUtilObject; only needed if you are assigning

Reply:
Thanks for calling this out. Removed global HUtilObject and tested with new changes